src/modules/dashboard/components/ColonyHome/ColonyExtensions/ColonyExtensions.tsx

```diff
@@ -63,7 +63,12 @@ const ColonyExtensions = ({ colony: { colonyName, colonyAddress } }: Props) => {
             })}
         </ul>
       ) : (
-        <SpinnerLoader />
+        <>
+          <SpinnerLoader />
+          <span className={styles.loadingText}>
+            <FormattedMessage {...MSG.loadingData} />
+          </span>
+        </>
```

Comment:
This is not correct.

Yes I know you just expanded on previous functionality, but now is a chance to fix it.

So this is not correct because if it doesn't find any extensions (and it might happen for non-fully deployed colonies), you will always show a spinner. This is not correct. We only show a spinner while actually loading, but if we don't find data, than we hide the spinner, and just don't show anything else.

Please change it to something like this:
```
if (loading) {
  return <Spinner ... >
}

return data && <Component ... >
```

Note that if we don't have any data, **don't** show the title at all, hide it.